test/scale/propagator_compliance_api_scale_test.go

```diff
@@ -0,0 +1,199 @@
+package scale
```

Comment:
Could you please consolidate this test in `case18`? Since it's quick to run, I don't see any harm including it in the normal e2e test package.